springwolf-ui/src/app/components/channels/channel-main/channel-main.component.ts

```diff
@@ -57,6 +60,15 @@ export class ChannelMainComponent implements OnInit {
       this.headersTextAreaLineCount = this.headersExample?.lineCount || 1;
       this.messageBindingExampleTextAreaLineCount =
         this.messageBindingExample?.lineCount || 1;
+
+      if (this.canPublishSubscription !== undefined) {
```

Comment:
Why do we need to unsubscribe?

Reply:
Because I dont know Angular/rxjs well enough.
You are right, http get is a finite value and therefore does not needs this
Source: https://stackoverflow.com/a/41177163